docs/api/model.rst

```diff
@@ -21,6 +21,22 @@ Language Modeling
     StandardRNN
     get_model
 
+Character Encoder
+-----------------
+
+.. autosummary::
+    :nosignatures:
+
+    CharacterEncoder
```

Comment:
Would it be appropriate to name it as ConvolutionEncoder? We don't have any module that dictates the form of the input data so far.

Reply:
this encoder might be useful to encode sequences of other granularity

Reply:
Yes, the encoder could be useful for both character matrix as well as sentence matrix.